extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -0,0 +1,172 @@
+			},
+			{
+				title: constants.MIGRATION_TIME_LABEL
+			}];
+
+		// create individual card component for each property in above list
+		headerCardsContainer.addItems(assessmentHeaderLabels.map(l => this.createCard(l)));
+
+		headerContainer.addItems([targetTypeContainer, headerCardsContainer]);
+
+		return headerContainer;
+	}
+
+	// function defining ui for card component in ui section.
+	private createCard(linkMetaData: IActionMetadata) {
+
+		const cardContainer = this._view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			width: 190,
+			height: 75
+		}).withProps({
+			CSSStyles: {
+				...styles.CARD_CSS,
+			}
+		}).component();
+
+		const cardHeading = this._view.modelBuilder.text().withProps({
+			value: linkMetaData.title,
+			CSSStyles: {
+				...styles.BODY_CSS,
+				'margin': '0px'
+			},
+		}).component();
+
+		const cardText = this._view.modelBuilder.text().withProps({
+			value: linkMetaData.value,
+			CSSStyles: {
+				...styles.LABEL_CSS,
+				'overflow-wrap': 'break-word'
+			},
+		}).component();
+
+		cardContainer.addItems([cardHeading, cardText]);
+		this._valueContainers.push(cardText);
+
+		return cardContainer;
+	}
+
+	// function to populate the values of properties displayed in the cards.
+	public async populateAssessmentDetailsHeader(migrationStateModel: MigrationStateModel): Promise<void> {
+		// this value is populated to handle the case when user selects a target type and want to resume later.
+		this._targetSelectionDropdown.value = this.getTargetTypeBasedOnModel(migrationStateModel._targetType);
+
+		const recommendedConfigurations = await utils.getRecommendedConfiguration(migrationStateModel._targetType, migrationStateModel);
+		let configurationValue = recommendedConfigurations[0] ?? "--";
+
+		if (migrationStateModel._targetType === MigrationTargetType.SQLVM && recommendedConfigurations?.length > 1) {
+			configurationValue = recommendedConfigurations[0] + "\n" + recommendedConfigurations[1];
+		}
+		const assessmentHeaderValues = [
+			{
+				value: configurationValue
+			},
+			{
+				value: String(migrationStateModel?._assessedDatabaseList.length)
+			},
+			{
+				value: String(migrationStateModel._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === migrationStateModel._targetType && issue.issueCategory === IssueCategory.Issue)?.length === 0)?.length)
+			}];
+
+		// iterating over each value container and filling it with the corresponding text.
+		let index = 0;
+		this._valueContainers.forEach((valueContainer) =>
+			valueContainer.value = assessmentHeaderValues[index++].value);
+	}
+
+	// function that create target selection dropdown
+	private createTargetTypeContainer(): azdata.FlexContainer {
+		const targetTypeContainer = this._view.modelBuilder.flexContainer().component();
+
+		const selectLabel = this._view.modelBuilder.text().withProps({
+			value: constants.SELECT_TARGET_LABEL + ":",
```

Comment:
please create a new resource string with ":" as part of the string instead of appending values

Reply:
done